lib/mldsa/vs_wasm.js

```
@@ -0,0 +1,769 @@
+    constructor(setup, id) {
+        const ptr0 = passArray8ToWasm0(setup, wasm.__wbindgen_malloc);
+        const len0 = WASM_VECTOR_LEN;
+        const ptr1 = passStringToWasm0(id, wasm.__wbindgen_malloc, wasm.__wbindgen_realloc);
+        const len1 = WASM_VECTOR_LEN;
+        const ret = wasm.keygensession_new(ptr0, len0, ptr1, len1);
+        if (ret[2]) {
+            throw takeFromExternrefTable0(ret[1]);
+        }
+        this.__wbg_ptr = ret[0] >>> 0;
+        KeygenSessionFinalization.register(this, this.__wbg_ptr, this);
+        return this;
```

Biome (2.4.4)

[error] The constructor should not return a value.

(lint/correctness/noConstructorReturn)